points/module.py

```diff
@@ -85,9 +85,8 @@ async def points_leaderboard(self, ctx):
         await utils.Discord.delete_message(ctx.message)
 
         if len(embeds) > 1:
-            scrollable_embed = utils.ScrollableEmbed()
-            scrollable_embed.from_iter(ctx, embeds)
-            await scrollable_embed.scroll(ctx)
+            scrollable_embed = utils.ScrollableEmbed(ctx, embeds)
```

Comment:
This if else is redundant due to your fix of empty / 1 item list

Reply:
Yeah, true. Fixed